files/js.js

```diff
@@ -0,0 +1,161 @@
+$(".inscription").click(function(){
+    $(".match_container").show();
+    $(".slack_container").hide();
+});
+
+$(".slackusers").click(function(){
+    $(".match_container").hide();
+    $(".slack_container").show();
+    pop();
+});
+
+$.ajaxSetup({
+    async: false
+});
+
+let slackUsers = null;
+$.getJSON("getSlackUsers.php", function( data ) {
+    slackUsers = data["members"];
+});
+
+function userprofileName(user) {
+    var name = "";
+    
+    if(user.profile.hasOwnProperty("first_name") && user.profile.first_name.length > 0 && 
+       user.profile.hasOwnProperty("last_name")  && user.profile.last_name.length > 0) {
+	name += user.profile.first_name + " " + user.profile.last_name;
+    } else if(user.profile.hasOwnProperty("real_name") && user.profile.real_name.length > 0) {
+	name += user.profile.real_name;
+    } else if(user.profile.hasOwnProperty("display_name") && user.profile.display_name.length > 0) {
+	name += user.profile.display_name;
+    } else if(user.hasOwnProperty("real_name")  && user.real_name.length > 0) {
+	name += user.real_name;
+    } else if(user.hasOwnProperty("display_name") && user.display_name.length > 0) {
+	name += user.display_name;
+    }
+
+    return name;
+}
+
+function pop() {
+    var slack_IDS = [];
+    $(".matched[slackID]").each(function(){
```

Comment:
could you please help me understand what this `.matched[slackID]` represents?
I mean: I see that `matched` is a class added to the `div` on which I click, but what does the added `[slackID]` means? (Is it for instance the attribute `slackid` of the divs with the class `matched` or am I completely misunderstanding this?)

Reply:
It selects elements (with class `.matched`) that have the specified attribute (`slackid`), with any value. 
https://api.jquery.com/has-attribute-selector/